atlas/schema.hcl

```
@@ -1,11 +1,476 @@
+    columns = [column.income_id, column.snapshot_id]
+  }
+  foreign_key "income_id" {
+    columns     = [column.income_id]
+    ref_columns = [table.income_v1.column.income_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+  foreign_key "scan_id" {
+    columns     = [column.scan_id]
+    ref_columns = [table.scan_v1.column.scan_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+}
+table "income_v1" {
+  schema = schema.public
+  column "project_id" {
+    null = false
+    type = bigint
+  }
+  column "income_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  primary_key {
+    columns = [column.project_id, column.income_id]
+  }
+  foreign_key "project_id" {
+    columns     = [column.project_id]
+    ref_columns = [table.project_v1.column.project_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+  index "income_id_idx" {
+    unique  = true
+    columns = [column.income_id]
+  }
+}
+
+table "organization_snapshot_v1" {
+  schema = schema.public
+  column "organization_id" {
+    null = false
+    type = bigint
+  }
+  column "snapshot_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  column "title" {
+    null = false
+    type = text
+    comment = "name of business"
+  }
+  column "income_type" {
+    null = false
+    type = text
+  }
+  column "industry_code" {
+    null = false
+    type = text
+  }
+  column "main_product" {
+    null = true
+    type = text
+    comment = "main product/service sold by the business"
+  }
+  column "partnership" {
+    null = false
+    type = bool
+    default = false
+  }
+  column "address" {
+    null = true
+    type = text
+    comment = "business physical address if applicable"
+  }
+  column "create_time" {
+    null = false
+    type = time
+  }
+  primary_key {
+    columns = [column.organization_id, column.snapshot_id]
+  }
+  foreign_key "organization_id" {
+    columns     = [column.organization_id]
+    ref_columns = [table.organization_v1.column.organization_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+}
+table "organization_v1" {
+  schema = schema.public
+  comment = "Refer to the link for explanations of the organization table fields: https://help.wealthsimple.com/hc/en-ca/articles/4408339655323-How-do-I-report-my-self-employment-income-on-a-T2125"
+  column "organization_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+ column "title" {
+    null = false
+    type = text
+    comment = "name of business"
+  }
+  column "income_type" {
+    null = false
+    type = text
+  }
+  column "industry_code" {
+    null = false
+    type = text
+  }
+  column "main_product" {
+    null = true
+    type = text
+    comment = "main product/service sold by the business"
```

If we're using full sentences and maintaining `organization` as the keyword instead of `business`.
```suggestion
    comment = "The main product/service sold by the organization."
```